extensions/amp-story/1.0/amp-story-system-layer.css

```diff
@@ -53,6 +55,54 @@
   opacity: 0 !important;
 }
 
+.i-amphtml-story-story-attribution {
+  top: 0 !important;
+  border: none !important;
+  pointer-events: auto !important;
+  cursor: pointer !important;
+  padding: 6px 3px 0 !important;
+  height: 48px !important;
+  place-items: center !important;
+  display: flex !important;
+  text-decoration: none !important;
+  overflow: hidden !important;
+  visibility: hidden !important;
+}
+
+[desktop] .i-amphtml-story-story-attribution {
+  padding: 6px 6px 0 !important;
+  left: 0 !important;
+  right: 0 !important;
+}
+
+.i-amphtml-story-story-attribution > * {
+  margin: 0px 3px !important;
+}
+
+.i-amphtml-story-attribution-logo-container {
+  display: grid !important;
+  place-items: center !important;
+}
+
+.i-amphtml-story-attribution-logo {
+  border-radius: 100% !important;
+  border: 1px solid #D4D4D4 !important;
+  width: 32px !important;
+  height: 32px !important;
+  object-fit: cover !important;
+  box-shadow: 0px 0px 6px rgba(0, 0, 0, 0.36) !important;
+}
+
+.i-amphtml-story-attribution-text {
```

Comment:
nit:
I wonder if we want a max width on this. 
It might be okay though since it takes up all available space then ellipsis. 
<img width="401" alt="Screen Shot 2021-04-02 at 10 39 55 AM" src="https://user-images.githubusercontent.com/3860311/113425476-e9692f80-939f-11eb-922d-471d186da135.png">

Reply:
Yeah, I got rid of it in 81372eccf9a909dac363bf48ab6c167fd97196f8 from the discussion in https://github.com/ampproject/amphtml/pull/33489#discussion_r603492478. Basically using a max-width would've been fragile since the controls could take more space at any given point (e.g. when embedded in a player and adding custom controls). Using a flex layout seems like a more robust solution since they will take the space they need naturally.